packages/ses/scripts/bundle.js

```diff
@@ -15,10 +16,11 @@ const write = async (target, content) => {
 };
 
 const main = async () => {
-  const bundle = await makeBundle(
+  const version = packageJson.version;
+  const bundle = `// v${version}\n${await makeBundle(
```

Comment:
Optional: I would like to move the await into its own statement and name the intermediate variable. Maybe this can be `versionedBundle`.

Agoric style encourages `await` to only be used as a top level expression or the right-side of an assignment in order to make the event delimiter obvious to reviewers. We sometimes even put large comments above the await so it doesn’t go unnoticed. A lot of the Agoric code operates in an adversarial environment and uses events as low-weight transactions. All code is responsible for maintaining its invariants between events so that events from an attacker don’t get interleaved between partial events. This is not a concern for this script, but a habit we like to encourage.

Reply:
@kriskowal What's the "event delimiter"?

Reply:
i initially split it to `versionedBundle` too, but reverted thinking too many invasive changes 😅 updated in https://github.com/endojs/endo/pull/1854/commits/bccf39aea1b0414102bbf13cce05fac6a545f28d

> @kriskowal What's the "event delimiter"?

`await` delimiter i believe (as opposed to template var backtick or dollar sign delimiters)